src/server/search/search_family_test.cc

```diff
@@ -190,4 +190,23 @@ TEST_F(SearchFamilyTest, Numbers) {
   */
 }
 
+TEST_F(SearchFamilyTest, TestLimit) {
+  for (unsigned i = 0; i < 20; i++)
+    Run({"hset", to_string(i), "match", "all"});
+  Run({"ft.create", "i1", "SCHEMA", "match", "text"});
+
+  // Default limit is 10
+  auto resp = Run({"ft.search", "i1", "all"});
+  EXPECT_THAT(resp, ArrLen(10 * 2 + 1));
```

Comment:
Don't you want to look at the content here as well?

Reply:
I don't, because as of now its not determined what will be returned